crm_request/models/request.py

```diff
@@ -172,3 +173,21 @@ def write(self, values):
                     'author_id': values['partner_id']
                 })
         return True
+
+
+class AssignRequestWizard(models.TransientModel):
+    _name = 'assign.request.wizard'
+    user_id = fields.Many2one('res.users', 'Assign to',
+                              default=lambda self: self.env.user,)
+    intern_note = fields.Text('Internal note')
+
+    @api.multi
+    def assign_to(self):
+        self.ensure_one()
+        model = self.env.context.get('active_model')
+        model_id = self.env.context.get('active_id')
+        request = self.env[model].browse(model_id)
+        request.user_id = self.user_id
+        if self.intern_note:
+            request.message_post(subject=u'Message pour ' + self.user_id.name,
```

Comment:
Never put french text in code. It should be English wrapped in the translate function _()